### ISSUE TITLE
W505 not triggered on multi-line docstring

### ISSUE THREAD

**Issue by Andy-Dihong-Luo** [charliermarsh/ruff#4344]
I've enabled W505 and set max-doc-length to 72. However, it didn't trigger when I exceed the limit on muti-line docstring. 
For example:
```python
def foo():
    """A VERY-LONG summary that exceed 72 characters."""  # W505 triggered as usual 
    return
```
However:
```python
def foo():
    """A short summary.
   
    A VERY-LONG description that exceed 72 characters.  # Nothing happened
    """
    return
```
I'd like to know if it's a bug or it's as designed. As instructed in PEP 8, docstrings should be wrapped at 72 characters, and Ruff is expected to warn that.

_Originally posted by @Andy-Dihong-Luo in https://github.com/charliermarsh/ruff/discussions/4264_

**Comment by charliermarsh** [comment]
Taking a look at this.

**Comment by charliermarsh** [comment]
Looks like a bug.
